cognite/powerops/resync/core/main.py

```diff
@@ -367,6 +368,18 @@ def apply(
     return written_changes
 
 
+def apply2(config_dir: Path, client: PowerOpsClient | None = None) -> None:
+    client = client or PowerOpsClient.from_settings()
+
+    importer = PowerAssetImporter.from_directory(config_dir / "production")
+
+    assets = importer.to_power_assets()
+
+    client.v1.upsert(assets)
+
+    print(f"Upserted {len(assets)} assets")
```

Comment:
**suggestion:** add some logging in addition

Reply:
It is a CLI and print is rich print, lets not do logging. The information for the user should go out on the terminal.

Reply:
Yes there are inconsistencies in the code, but lets leave that. Feel free to add a Jira task on it.